package.json

```diff
@@ -8,6 +8,9 @@
   "documentationUrl": "https://github.com/segmentio/analytics-csharp",
   "changelogUrl": "https://github.com/segmentio/Analytics-CSharp/releases",
   "licensesUrl": "https://github.com/segmentio/analytics-unity/blob/main/LICENSE.md",
+  "dependencies": {
+    "com.unity.nuget.newtonsoft-json": "3.0.2"
```

Comment:
can we add a comment here to mention the version should be corresponding to newtonsoft version used in serialzation.net? the correspondence can be found in unity's doc [here](https://docs.unity3d.com/Packages/com.unity.nuget.newtonsoft-json@3.0/manual/index.html). so when we update the version, we need to update here and serialzation.net

Reply:
Discussed offline, no comments in json, we will likely add something about this to Serialization.NET